libraries/AP_Camera/AP_Camera_Logging.cpp

```diff
@@ -64,12 +65,20 @@ void AP_Camera_Backend::Write_CameraInfo(enum LogMessages msg, uint64_t timestam
 void AP_Camera_Backend::Write_Camera(uint64_t timestamp_us)
 {
     Write_CameraInfo(LOG_CAMERA_MSG, timestamp_us);
+    auto *mount = AP_Mount::get_singleton();
+    if (mount!= nullptr) {
+        mount->Write_Mount(LOG_MOUNT_MSG, timestamp_us);
```

Comment:
This looks pretty good.  We should be specifying the instance though... which should match the camera instance.  This is built on the assumption that camera1 is in mount1, camera2 is in mount2.  Eventually we will need to fix this assumption and allow users to specify which mount the camera is in, but it's OK for now.